packages/worker/src/debug-replayer/worker-thread.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Worker thread entrypoint used by ./index.ts to synchronously make an HTTP call to the "runner".
```

Comment:
Why a thread? I think a `docs/vs-code-debugger.md` architecture description would be helpful

Reply:
Definitely agree a diagram will help here.
There's another comment saying why we use a thread.

Reply:
because you need to block the main thread?